test/lib/session-test.js

```diff
@@ -53,7 +53,7 @@ describe('Keystone.session', function() {
 				sinon.assert.calledOnce(req.session.regenerate);
```

Comment:
All changes in this file are unrelated to this PR.  Please extract them and move to another PR.